root/static/scripts/edit/MB/CoverArt.js

```diff
@@ -159,7 +159,7 @@ MB.CoverArt.upload_status_enum = {
 
 MB.CoverArt.validate_file = function (file) {
     var deferred = $.Deferred();
-    var reader = new FileReader();
+    var reader = new window.FileReader();
```

Comment:
We have to be explicit now about things that exist on `window` as opposed to the global object (which may not be `window`). Which seems like a good idea anyway.